crates/api_models/src/admin.rs

```diff
@@ -760,6 +915,102 @@ pub struct MerchantConnectorInfo {
 }
 
 /// Response of creating a new Merchant Connector for the merchant account."
+#[cfg(all(feature = "v2", feature = "merchant_connector_account_v2"))]
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+#[serde(deny_unknown_fields)]
+pub struct MerchantConnectorResponse {
+    /// Type of the Connector for the financial use case. Could range from Payments to Accounting to Banking.
+    #[schema(value_type = ConnectorType, example = "payment_processor")]
+    pub connector_type: api_enums::ConnectorType,
+    /// Name of the Connector
+    #[schema(value_type = Connector, example = "stripe")]
+    pub connector_name: String,
+
+    /// A unique label to identify the connector account created under a business profile
+    #[schema(example = "stripe_US_travel")]
+    pub connector_label: Option<String>,
```

Comment:
This will be a display name and always in response we can have value for this right?

Reply:
Yes, but it's an optional field in diesel_models too. Can make this as mandatory in next PR